packages/api/src/jobs/send_email.ts

```diff
@@ -0,0 +1,26 @@
+import { env } from '../env'
+import { sendWithMailJet } from '../services/send_emails'
+import { sendEmail } from '../utils/sendEmail'
+
+export const SEND_CONFIRMATION_EMAIL_JOB = 'send-confirmation-email'
+
+export interface SendConfirmationEmailData {
+  emailAddress: string
+  link: string
+  templateData: Record<string, any>
+}
+
+export const sendConfirmationEmail = async (
```

Comment:
Actually i think a better name is AddressVerificaction. So `sendAddressVerificationEmail`

Reply:
@jacksonh Yeah I have another function called `sendVerificationEmail` and it is used for updating email address so Should I rename that one too?

Reply:
Hah yeah i think our names should be:

- SendPasswordResetEmail
- SendPasswordChangeEmail 
- SendNewAccountVerificationEmail

Is that better? I guess account verification and confirmation are similar. I would still add the Account part at least, as it is hard to tell what we are confirming like it could be "confirming reset, confirming sign up, confirming change"

Reply:
It is way better.

Only `SendPasswordChangeEmail` probably is better `SendEmailChangeEmail`? It sounds weird though

Reply:
We send an email verification to the user if they changed the registered email address